public/app/features/dashboard/dashgrid/PanelLinks.tsx

```diff
@@ -47,6 +47,8 @@ export function PanelLinks({ panelLinks, onShowPanelLinks }: Props) {
 const getStyles = (theme: GrafanaTheme2) => {
   return {
     menuTrigger: css({
+      height: '100%',
```

Comment:
🤔 adding the height of 100% is making the `ToolbarButton` not preserve the 32px.

![image](https://user-images.githubusercontent.com/239999/214542430-4dc97ce1-5fdc-41b9-a466-10593a19fea9.png)

Reply:
@kaydelaney I double check and this problem still present, I think we can remove that height 100% 😄